components/lib/VCarousel.vue

```diff
@@ -2,13 +2,10 @@
   <div class="slider">
     <transition-group :name="'slide-' + transitionName" tag="div">
       <div :key="currentSlide" ref="carousel" class="slide">
-        <slot />
+        <slot name="slides" />
       </div>
     </transition-group>
-    <v-stack class="icons" justify="center">
-      <v-image path="icons/arrow-left.svg" @click.stop="changeSlide(-1)" />
-      <v-image path="icons/arrow-right.svg" @click.stop="changeSlide(1)" />
-    </v-stack>
+    <slot />
```

Comment:
I don't think we need this slot. Why not just place the slider icons below the carousel?